### PR TITLE
Add a use case with push events for tags

### PR DESCRIPTION
**Pull request by dmarcoux**
To test this locally:
1. Pull the changes from this PR, then run `daps2docker .`
2. Open the generated user guide in your favorite browser: `firefox /tmp/daps2docker-SXo2tVAc/obs-user-guide/html/book.obs-user/cha.obs.scm_ci_workflow_integration.html` (the string after `daps2-docker-` will be different).

Preview of the changes:
preview-1
preview-2

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Add a use case with push events for tags (M)](https://trello.com/c/fVudlOMl/1858-add-a-use-case-with-push-events-for-tags-m)

**Review by saraycp — APPROVED** [review]

File: xml/obs_scm_ci_workflow_integration.xml [hunk 1]
```diff
@@ -1353,5 +1353,128 @@ workflow:
...
+        source_project: home:jane
+        source_package: my_package
+        target_project: home:jane:releases
+  filters:
+    event: tag_push</screen>
+
+      <note>
+        <para>A Git tag is a pointer to a commit, and commits exist independently of branches. This is why
+        filtering on branches is <emphasis>not</emphasis> possible when working with push events for tags.</para>
+      </note>
```
Comment (hennevogel) [inline xml/obs_scm_ci_workflow_integration.xml:1399]: Why mention this?

**Review by hennevogel — COMMENTED** [review]
Overall I'm missing a bit and explanation what will happen if I do all of this....

File: xml/obs_scm_ci_workflow_integration.xml (hunk 1 above)
Comment (dmarcoux) [reply xml/obs_scm_ci_workflow_integration.xml:1399]: I'm expecting questions about this. Should we remove this note?

Comment (dmarcoux) [reply xml/obs_scm_ci_workflow_integration.xml:1399]: I think it won't be really common, so I'll remove it.

**Comment by dmarcoux** [comment]
I've pushed again, please have another look.

**Review by saraycp — APPROVED** [review]
I think the first paragraph is more understandable now.
The rest, being more concise, are easier to read as well.

**Comment by hennevogel** [comment]
I'm still missing description of the outcome of this, all of this seems like a very general technical description to me.

This is basically the only sentence we provide for our customers to decide if this is the *right* thing for them:

> Once the workflow is in place, every tag you push to your SCM repository will create a package on OBS for the source code associated to the tag's commit.

Where will this package be created? How will it look like? How can my packages users benefit from this? Is there something else I need to do?

**Comment by hennevogel** [comment]
> Please have another look

Maybe I have missed something but I don't see any changes?

**Comment by dmarcoux** [comment]
Here's the [latest diff](https://github.com/openSUSE/obs-docu/compare/3bcdf6701f0767f4e49ce4e8d3d01d05eb846307..9ed22f88c51f363cc2ca36f71eeb989ea0bcb23e).

With the current changes, I'm explaining why someone would want to set this up, while also giving details on how this works. I've removed a lot of extra and superfluous information from the text I wrote when I first submitted this PR. I believe that I have answered your previous questions: `Where will this package be created? How will it look like? How can my packages users benefit from this? Is there something else I need to do?`.

**Review by danidoni — APPROVED** [review]

**Comment by danidoni** [comment]
> Here's the [latest diff](https://github.com/openSUSE/obs-docu/compare/3bcdf6701f0767f4e49ce4e8d3d01d05eb846307..9ed22f88c51f363cc2ca36f71eeb989ea0bcb23e).
> 
> With the current changes, I'm explaining why someone would want to set this up, while also giving details on how this works. I've removed a lot of extra and superfluous information from the text I wrote when I first submitted this PR. I believe that I have answered your previous questions: `Where will this package be created? How will it look like? How can my packages users benefit from this? Is there something else I need to do?`.

LGTM!

**Pull request merged**